packages/flutter/test/material/dialog_test.dart

```diff
@@ -243,6 +251,26 @@ void main() {
     expect(await result, equals(42));
   });
 
+  testWidgets('Custom padding on SimpleDialogOption', (WidgetTester tester) async {
+    const EdgeInsets customPadding = EdgeInsets.all(10);
+    final SimpleDialog dialog = SimpleDialog(
+      title: const Text('Title'),
+      children: <Widget>[
+        SimpleDialogOption(
+          onPressed: () {},
+          child: const Text('First option'),
+        ),
+      ],
+    );
+
+    await tester.pumpWidget(_appWithSimpleDialog(tester, dialog));
+    await tester.tap(find.text('X'));
+    await tester.pumpAndSettle();
+
+    final SimpleDialogOption dialogOption = tester.renderObject(find.byType(SimpleDialogOption));
+    expect(dialogOption.padding, customPadding);
```

Comment:
Instead of checking the `padding` property here, check to see if the contents of `DialogOption` are properly padded. There are similar tests [here](https://github.com/flutter/flutter/blob/c92fe1485f6bd991ed68885c0bb573b5676281e9/packages/flutter/test/material/dialog_test.dart#L404) and [here](https://github.com/flutter/flutter/blob/master/packages/flutter/test/material/list_tile_test.dart#L421). Let me know if you need any more guidance on this

Reply:
@shihaohong, sorry for the wait to update this PR, but I've applied [these changes](https://github.com/flutter/flutter/pull/50035/commits/9b1325e16622e7d140af23884468655ea5456d69#diff-e75a5bc47aaacb0ba2973b4f42a0bcd2R263).

Is that what you suggested?

Reply:
Yeah, that's much better! I left a comment that will make the test case better